### PR TITLE
Dsys 5 slider number input

### PR DESCRIPTION
**Pull request by nathanbrud**
## What this PR does 
Adds UI for arrows up/down for the slider editable input

## Screenshots & Recordings 

https://github.com/vimeo/iris/assets/9106375/6eb41187-762a-4c0b-a930-d1036921b2f5

## How it does that 
* Separate the EditableInput into a component
* Added Arrows component
* Handle click up/down logic

## Testing 
* Hover editable input in the slider
* See arrows up and down
* Click on them and see value is changing

### CODE REVIEW

File: src/components/inputs/Slider/SliderInputArrow.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    height="10"
+    viewBox="0 0 12 10"
+    fill="none"
+    {...props}
+  >
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M6.00178 4.91424L3.70689 7.2074L2.29321 5.79265L5.29547 2.79265C5.48308 2.60519 5.73747 2.49993 6.00269 2.50002C6.2679 2.50012 6.52222 2.60558 6.70968 2.79318L9.70743 5.79318L8.29268 7.20687L6.00178 4.91424Z"
+      fill="#11191D"
```
Comment (juliewongbandue) [inline src/components/inputs/Slider/SliderInputArrow.tsx:16]: I don't see the arrows in dark mode, that could probably be fixed with assigning this as `currentColor` and modifying the `color` in the `<ArrowsContainer>`.

[the remark this answers is not in the document]
Comment (nathanbrud) [reply src/components/inputs/Slider/SliderInputArrow.tsx:16]: done

[the remark this answers is not in the document]
Comment (juliewongbandue) [reply src/components/inputs/Slider/SliderInputArrow.tsx:16]: Sorry for the confusion, I meant this line here to be changed to `currentColor`. The color of the path won't be able to be modified from its parent if the `fill` is still a hardcoded color.

[the remark this answers is not in the document]
Comment (nathanbrud) [reply src/components/inputs/Slider/SliderInputArrow.tsx]: Changed it, it wasn't you, it was my confusion, sorry about that 🤣

**Review by juliewongbandue — APPROVED** [review]

**Pull request merged**